crypto/ml_dsa/ml_dsa_sign.c

```diff
@@ -47,8 +47,10 @@ static void signature_init(ML_DSA_SIG *sig,
  * @returns 1 on success and 0 on failure.
  */
 static int ml_dsa_sign_internal(const ML_DSA_KEY *priv, int msg_is_mu,
-                                const uint8_t *encoded_msg,
-                                size_t encoded_msg_len,
+                                const uint8_t *context_msg,
```

Comment:
Maybe update the comment above..
Instead of passing M` we pass it as 2 separate components
i.e. The encoded context as 01||(sizeof(ctx)||ctx and the msg

Reply:
Same comments apply to the verify_internal()